resource_customizations/apps/DaemonSet/actions/discovery.lua

```diff
@@ -1,3 +1,4 @@
 local actions = {}
 actions["restart"] = {}
+actions["scale"] = {["defaultValue"] = tostring(obj.spec.replicas), ["hasParameters"] = true,  ["errorMessage"] = "Enter any valid number", ["regexp"]= "^[0-9][0-9]*$"}
```

Comment:
Is `+` not supported? 


I'd expect a space before `=`.

Is `\d` defined to more than `[0-9]`?

Reply:
Added space and Updated regex